mmv1/products/vertexai/Schedule.yaml

```diff
@@ -0,0 +1,323 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: Schedule
+min_version: beta
+base_url: projects/{{project}}/locations/{{location}}/schedules
+create_url: projects/{{project}}/locations/{{location}}/schedules
+self_link: 'projects/{{project}}/locations/{{location}}/schedules/{{name}}'
+update_verb: :PATCH
+update_mask: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/vertex-ai/docs'
+  api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.schedules'
+async: !ruby/object:Api::OpAsync
+  actions:
+    - create
+    - delete
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+description:
+  'An instance of a Schedule periodically schedules runs to make
+  API calls based on user specified time specification and API
+  request type.'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name:
+      'vertex_ai_schedule'
+    primary_resource_id: 'schedule'
+    vars:
+      name: 'vertex_ai_schedule'
+      project: 'vertex-ai'
+      display_name: 'display-name'
+      max_run_count: 1
+      max_concurrent_run_count: 1
+      allow_queueing: true
+      cron: '0 0 * * *'
+      parent: '' # TODO
```

Comment:
not sure how to implement a `parent` field properly - presume it can be inferred from location and project somehow?

Reply:
What does the parent field do in this scenario?

Reply:
The parent field describes the project and region where the PipelineJob should be created. It has the format `projects/{project}/locations/{location}`